gateway/README.md

```diff
@@ -19,25 +19,76 @@ useful to use if you have a large bot in over 1000 or 2000 guilds.
 
 ## Features
 
-`twilight-gateway` includes only a feature: `simd-json`.
+### Deserialization
+
+`twilight-gateway` supports [`serde_json`] and [`simd-json`] for
+deserializing and serializing events. These features are mutually exclusive.
+`serde_json` is enabled by default.
+
+#### `serde_json`
+
+[`serde_json`] is the inverse of `simd-json` and will use the `serde_json`
+crate to deserialize responses.
+
+This is enabled by default.
+
+#### `simd-json`
+
+The `simd-json` feature enables [`simd-json`] support to use simd features of
+the modern cpus to deserialize responses faster. It is not enabled by
+default.
+
+To use this feature you need to also add these lines to
+`<project root>/.cargo/config`:
 
-`simd` feature enables [simd-json] support to use simd features of the modern cpus
-to deserialize json data faster. It is not enabled by default since not every cpu has those features.
-To use this feature you need to also add these lines to a file in `<project root>/.cargo/config`
 ```toml
 [build]
 rustflags = ["-C", "target-cpu=native"]
 ```
-you can also use this environment variable `RUSTFLAGS="-C target-cpu=native"`. If you enable both
-`serde_json` and `simd-json` at the same time; this crate uses `simd-json`. But it is recommended to
-disable `serde_json` if you are going to use `simd-json`. It is easy to switch to out:
+
+You can also set the environment variable
+`RUSTFLAGS="-C target-cpu=native"`. If you enable both `serde_json` and
+`simd-json` at the same time, then `simd-json` will be used.
+
+To enable `simd-json`, do something like this in your `Cargo.toml`:
 
 ```toml
 [dependencies]
-twilight-gateway = { branch = "trunk", default-features = false, features = ["simd-json"], git = "https://github.com/twilight-rs/twilight" }
+twilight-gateway = { branch = "trunk", default-features = false, features = ["native", "simd-json"], git = "https://github.com/twilight-rs/twilight" }
```

Comment:
should this be rustls?

Reply:
rather is simd json compatible with rustls?

Reply:
It should be Rustls, I've updated it. TLS features have no impact on serialization features